docs/howto/how-to-build-runtime.md

```diff
@@ -1,9 +1,226 @@
+
+```
+$ export BUILD_TYPE=release
+$ make install
+```
+
+Or you can simply do something like this:
+
+```
+$ BUILD_TYPE=release make install
+```
+
+The build method described here is a `native build` in which the build environment and execution environment are same. So, this command creates a runtime binary targeting the current build architecture, probably x86_64, as the execution environment. You can find the build output in the ./Product folder as follows:
+
+```
+$ tree -L 2 ./Product
+./Product
+├── obj -> /home/sjlee/star/one/Product/x86_64-linux.debug/obj
+├── out -> /home/sjlee/star/one/Product/x86_64-linux.debug/out
+└── x86_64-linux.debug
+    ├── BUILD
+    ├── CONFIGURE
+    ├── INSTALL
+    ├── obj
+    └── out
+
+5 directories, 3 files
+
+$ tree -L 3 ./Product/out
+./Product/out
+├── bin
+│   ├── nnapi_test
+│   ├── nnpackage_run
+│   ├── tflite_loader_test_tool
+│   └── tflite_run
+├── include
+│   ├── nnfw
+│   │   ├── NeuralNetworksEx.h
+│   │   ├── NeuralNetworksExtensions.h
+│   │   ├── NeuralNetworks.h
+│   │   ├── nnfw_dev.h
+│   │   └── nnfw.h
+│   └── onert
+│       ├── backend
+│       ├── compiler
+│       ├── exec
+│       ├── ir
+│       ├── misc
+│       └── util
+├── lib
+│   ├── libbackend_cpu.so
+│   ├── libcircle_loader.so
+│   ├── libneuralnetworks.so
+│   ├── libnnfw-dev.so
+│   ├── libnnfw_lib_benchmark.so
+│   ├── libnnfw_lib_misc.a
+│   ├── libonert_core.so
+│   └── libtflite_loader.so
+├── tests
+│   ├── FillFrom_runner
+│   ├── nnpkgs
+│   │   └── FillFrom
+│   └── scripts
+│       ├── benchmark_nnapi.sh
+│       ├── benchmark_nnpkg.sh
+│       ├── common.sh
+│       ├── framework
+│       ├── list
+│       ├── print_to_json.sh
+│       ├── test-driver.sh
+│       ├── test_framework.sh
+│       ├── test_scheduler_with_profiling.sh
+│       └── unittest.sh
+├── unittest
+│   ├── nnapi_gtest
+│   ├── nnapi_gtest.skip
+│   ├── nnapi_gtest.skip.noarch.interp
+│   ├── nnapi_gtest.skip.x86_64-linux.cpu
+│   ├── test_compute
+│   ├── test_onert
+│   ├── test_onert_backend_cpu_common
+│   ├── test_onert_frontend_nnapi
+│   └── tflite_test
+└── unittest_standalone
+    └── nnfw_api_gtest
+
+19 directories, 36 files
+
+```
+
+Here, let's recall that the main target of our project is the arm architecture. If you have a development environment running Linux for arm on a device made of an arm CPU, such as Odroid-XU4, you will get a runtime binary that can be run on the arm architecture with the same command above. This is the simplest way to get a binary for an arm device. However, in most cases, native builds on arm devices are too impractical as they require too long. Therefore, we will create an executable binary of an architecture other than x86_64 through a `cross build`. For cross-build method for each architecture, please refer to the corresponding document in the following section, [How to cross-build runtime for different architevture](#how-to-cross-build-runtime-for-different-architevture).
```

Comment:
[How to cross-build runtime for different architevture](#how-to-cross-build-runtime-for-different-architevture).
architevutre looks like typo.

Reply:
@dev-hue , Thank you! Would you please review with diff suggestion for easy fix? :)

Reply:
@lemmaa I didn't know that! Very useful! Thanks!
But, I miss-clicked the *commit suggestion* 😢.
I googled the way to revert it, I can't find...
Is there any way to revert..?